### PR TITLE
Extracted locales into seperate locales file on the rails way

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by waynepallen** [comment]
-1  we can't change this at this time because this barclamp is shared with hadoop 2.4 branch.

**Comment by tboerger** [comment]
Even if it is shared, it doesn't hurt. Everything should still work fine if this PR gets into hadoop 2.4 too: https://github.com/crowbar/crowbar/pull/1961

**Comment by arkadykanevsky** [comment]
-1 this change will break other branches that use this repo.

**Comment by tboerger** [comment]
@arkadykanevsky Where should it break anything? This is default rails behavior and would work like before.

**Comment by dirkmueller** [comment]
@tboerger rebase please

**Comment by tboerger** [comment]
Rebased.

File: crowbar_framework/config/locales/dell_raid.en.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+en:
+  raid:
+    SingleRaid10: Single Raid 10
+    JBODOnly: JBOD Only
+  barclamp:
+    dell_raid:
+      edit_deployment:
+        deployment: Deployment
+      edit_attributes:
+        attributes: Attributes
```
Comment (vuntz) [inline crowbar_framework/config/locales/dell_raid.en.yml]: I don't think we need the two attributes and deployment strings, do we?

Comment (tboerger) [inline crowbar_framework/config/locales/dell_raid.en.yml]: Fixed.

**Comment by tboerger** [comment]
Can get merged now :)

**Pull request merged**